airflow/providers/microsoft/azure/log/wasb_task_handler.py

```diff
@@ -176,19 +178,18 @@ def wasb_write(self, log: str, remote_log_location: str, append: bool = True) ->
         :type log: str
         :param remote_log_location: the log's location in remote storage
         :type remote_log_location: str (path)
-        :param append: if False, any existing log file is overwritten. If True,
+        :param append: if False, any existing log file is overwritten with the new log. If True,
```

Comment:
What happens when:

1) log file exists
2) Append = false
3) Overwrite = false

I think it's not really obvious and help is not helpful. Could we clarify that in the description of the parameters ?
?

Reply:
@potiuk yeah, np, will post asap here. thx

Reply:
@potiuk 
Is okay now?
```
        :param append: if False and overwrite = True, any existing log file is overwritten with the new log.
                       If True and overwrite = True, the new log is appended to any existing logs.
                       If True and overwrite = False the method will fail
        :type append: bool
        :param overwrite: if False, does not overwrite log file. If the file already exists the method will fail
                          if True, any existing log file is overwritten.
        :type overwrite: bool
```

Reply:
Much better !